### PR TITLE
Update polish.xml

### PR DESCRIPTION
**Pull request by przemo20**

### CODE REVIEW

**Comment by donho** [comment]
@ArkadiuszMichalski 
Could you validate the modification please?

File: PowerEditor/installer/nativeLang/polish.xml [hunk 1]
```diff
@@ -1258,41 +1258,39 @@ Kontynuować?"/> <!-- HowToReproduce: when you openned file is modified and save
...
-zostanie zamknięty i przeniesiony do Kosza.
+zostanie zamknięty i przeniesiony do kosza.
 Kontynuować?"/>
             <NoBackupDoSaveFile title="Zapisywanie" message="Nie udało się odnaleźć kopii zapasowej na dysku. Została usunięta.
 Żeby nie stracić danych, należy zapisać bieżące zmiany.
 Zapisać plik „$STR_REPLACE$”?"/>
-            <DoReloadOrNot title="Przeładowywanie" message="Plik „&quot;$STR_REPLACE$”
+            <DoReloadOrNot title="Przeładowywanie" message="Plik „$STR_REPLACE$” został zmodyfikowany przez inny program.
 
-został zmodyfikowany przez inny program.
```
Comment (ArkadiuszMichalski) [inline PowerEditor/installer/nativeLang/polish.xml:1286]: @przemo20 Please don't change the number of lines relative to the original english file. Revert to the previous variant or use another sentence (but keep the number of lines).

File: PowerEditor/installer/nativeLang/polish.xml [hunk 2]
```diff
@@ -1258,41 +1258,39 @@ Kontynuować?"/> <!-- HowToReproduce: when you openned file is modified and save
...
 Przeładować go?"/>
-            <DoReloadOrNotAndLooseChange title="Przeładowywanie" message="Plik „$STR_REPLACE$”
+            <DoReloadOrNotAndLooseChange title="Przeładowywanie" message="Plik „$STR_REPLACE$” został zmodyfikowany przez inny program.
 
-został zmodyfikowany przez inny program.
```
Comment (ArkadiuszMichalski) [inline PowerEditor/installer/nativeLang/polish.xml:1290]: @przemo20 Please don't change the number of lines relative to the original english file. Revert to the previous variant or use another sentence (but keep the number of lines).

**Comment by ArkadiuszMichalski** [comment]
@donho After merging this I will update `polish.xml` according to the latest commit.

**Review by ArkadiuszMichalski — APPROVED** [review]

**Pull request closed without merging**